packages/core/src/release.ts

```diff
@@ -283,6 +283,7 @@ export default class Release {
    * @param from - Tag or SHA to start at
    * @param to - Tag or SHA to end at (defaults to HEAD)
    */
+  @memoize()
```

Comment:
I was iffy on adding memoization here. But since `auto` will never care about generated commits I **think** it's safe to memoize this function. This should also help with some log reduction